src/main/webapp/js/common.js

```diff
@@ -484,20 +484,22 @@ function getPointValue(s, ditchZero) {
 /**
  * Checks if element is within browser's viewport.
  * @return true if it is within the viewport, false otherwise
+ * @see http://stackoverflow.com/q/123999
  */
 function isWithinView(element) {
```

Comment:
It is indeed used for testing by `/src/test/java/teammates/test/pageobjects/AppPage.java`